### ISSUE TITLE
Plugins: Running `./gradlew run` in external plugins fails

### ISSUE THREAD

**Issue by spinscale** [elastic/elasticsearch#32807]
**Elasticsearch version** (`bin/elasticsearch --version`): 6.3.2 (all of 6.3 is affected), works in 6.2 and below

When trying to call `./gradlew run` in an external plugin, the following exceptions pops up

```
./gradlew run
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :
=======================================
Elasticsearch Build Hamster says Hello!
=======================================
  Gradle Version        : 4.7
  OS Info               : Mac OS X 10.13.6 (x86_64)
  JDK Version           : Oracle Corporation 10.0.1 [Java HotSpot(TM) 64-Bit Server VM 10.0.1+10]
  JAVA_HOME             : /Users/alr/.sdkman/candidates/java/10.0.1-oracle
  Random Testing Seed   : CDFB648806AD1486

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':run#extract'.
> Could not resolve all files for configuration ':run_elasticsearchDistro'.
   > Could not find org.elasticsearch.distribution.zip:elasticsearch:6.3.2.
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/org/elasticsearch/distribution/zip/elasticsearch/6.3.2/elasticsearch-6.3.2.pom
         https://repo.maven.apache.org/maven2/org/elasticsearch/distribution/zip/elasticsearch/6.3.2/elasticsearch-6.3.2.zip
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.7/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 10s
```

It seems that the correct artifact would be the `elasticsearch-oss` distribution at https://repo.maven.apache.org/maven2/org/elasticsearch/distribution/zip/elasticsearch-oss/6.3.2/elasticsearch-oss-6.3.2.zip

If you want to reproduce you can run

```
# this will create an ingest plugin skeleton in the ingest-awesome directory
cookiecutter --no-input gh:spinscale/cookiecutter-elasticsearch-ingest-processor
cd ingest-awesome
./gradlew run
```

**Comment by rjernst** [comment]
I think this has already been addressed by https://github.com/elastic/elasticsearch/pull/32549?

**Comment by spinscale** [comment]
indeed, fixed in 6.4.0
